README.md

```diff
@@ -14,52 +120,110 @@ ingestion projects and optionally pushing them to a storage endpoint.
+>    gitRemoteUrl: https://github.com/JupiterOne/graph-google-cloud.git
+>    config:
+>      SERVICE_ACCOUNT_KEY_FILE={}
+>      PROJECT_ID="..."
+>      ORGANIZATION_ID="..."
+>      CONFIGURE_ORGANIZATION_PROJECTS=false
+> storage:
+>  engine: neo4j
+>  config: 
+>    username: neo4j
+>    password: devpass
+>    uri: bolt://localhost:7687
+> ```
 
 ### Running Starbase
 
-1. Perform `yarn starbase setup` to perform a clone or update of all integrations
+Starbase exposes a CLI for bootstrapping graph integration
+development and execution.
+
+```
+❯ yarn starbase --help
+
+Usage: yarn starbase [options] [command]
+
+Starbase graph ingestion orchestrator
+
+Options:
+  -h, --help      display help for command
+
+Commands:
+  run             collect and upload entities and relationships
+  setup           clone repositories listed in config.yaml
+  help [command]  display help for command
+```
+
+1. Run `yarn starbase setup` to clone or update all integrations
    listed in the `config.yaml` file as well as install all dependencies for each
    integration.
-2. Perform `yarn starbase run` to collect data for each listed integration and
-   then push collected data if a storage endpoint has been listed in
-   `config.yaml`.
+2. Run `yarn starbase run` to collect data for each listed integration and
+   then push collected data to the storage endpoint listed in `config.yaml`.
 
 For additional information on using Neo4j as a storage endpoint, please see the
 [README.md](docker/README.md) provided.
 
-### Making Contributions
+### Contributing
+
+Starbase is composed of three components:
 
-Start by taking a look at the source code. Each integration is basically a set
-of functions called steps, each of which ingests a collection of resources and
-relationships. The goal is to limit each step to as few resource types as
-possible so that should the ingestion of one type of data fail, it does not
-necessarily prevent the ingestion of other, unrelated data. That should be
-enough information to allow you to get started coding!
+1. **Starbase Core**
+  
+    The Starbase core project is an orchestration engine that handles bootstraping the underlying graph integrations.
 
-See the
-[SDK development documentation](https://github.com/JupiterOne/sdk/blob/main/docs/integrations/development.md)
-for a deep dive into the mechanics of how integrations work.
+2. **Graph Integrations**
 
-See the [docs/development.md](docs/development.md) in any of the existing
-integrations for any additional details about developing other integrations.
+    These are the tools that perform data collection from third party systems and services. You can find a full list of 
+    supported [graph integrations here](https://github.com/jupiterone?q=graph-&type=all&language=&sort=). If you have a feature request, 
+    a bug to report, or you'd like to contribute to one of the supported integrations, please navigate to the specific integration repository.
```

Comment:
Does this mean we're going to take care to manage Issues on GitHub?

Reply:
Imo, yes. Seems the most natural.